cypress/utils/utils.ts

```diff
@@ -1035,29 +1035,26 @@ export function deleteByList<T extends Deletable>(array: T[]): void {
     });
 }
 
-export function deleteAllStakeholders(cancel = false): void {
+export function deleteAllStakeholders(): void {
     Stakeholders.openList();
-    selectItemsPerPage(100);
-    cy.wait(0.5 * SEC);
-    cy.get(commonView.appTable)
+    cy.get("table[aria-label='Stakeholders table']", { timeout: 2 * SEC })
```

Comment:
Is it not same as commonView.Table ?

Is yes , then this method can be used to delete any page and all delete methods can be replaced by this.
Like deleteAllApplications(), deleteAllJobFunctions() etc .

WDYT?

Reply:
It is not, that’s why I had to change it.

Other tables have aria-label=main-table and thats the value of commonView.table.

This one is aria-label=Stakeholders Table